package.json

```diff
@@ -36,6 +36,8 @@
     "bookshelf": "^0.14.2",
     "hapi": "16.6.1",
     "hapi-bookshelf-serializer": "2.1.0",
+    "hapi-format-error": "^2.1.0",
+    "joi": "12",
```

Comment:
We might want a pessimistic version constraint, for some flexibility:
```suggestion
    "joi": "^12.0.0",
```

Reply:
Docs say node 6 is unsupported in later versions of Joi?
![image](https://user-images.githubusercontent.com/5596573/58725893-21168300-8395-11e9-96cb-17dc5412241d.png)

Reply:
Oh that should be okay for this repo - looks like the `.node-version` is `8.11.3`. Also this would allow upgrades from `12.0.0` up to and excluding `12.1.0`, so theres less of a chance of a breaking change related to node (if they're using somewhat sane versioning practices).